packages/components/src/Button/button.tsx

```diff
@@ -15,6 +15,13 @@ export type ButtonProps = ButtonHTMLElementProps & {
   variant?: ClickableProps<"button">["variant"];
 };
 
+/**
+ * Component for making buttons that do not navigate the user to another page.
+ *
+ * This component is called Button because it should be used to making button elements;
+ * however, it can be styled to look like a link. In terms of the look and feel of the
+ * component in the UI, the Button and Link components are exactly the same.
+ */
```

Comment:
Added a docstring to the `Button` component that's a mirror image of the one on the `Link`.